CI/travis.set-build-info.sh

```diff
@@ -15,7 +15,9 @@ if [ -z "${TRAVIS_TAG}" ]; then
   else
     if [ "${MUDLET_VERSION_BUILD}" = "-ptb" ]; then
       DATE=$(date +'%Y-%m-%d')
-      MUDLET_VERSION_BUILD="${MUDLET_VERSION_BUILD}-${DATE}"
+      # add a short commit to version for changelog generation know  what was last released
+      SHORT_COMMIT=$(echo "${COMMIT}" | cut -c1-5)
```

Comment:
Same thing here, as `${COMMIT}` does not seem to be defined before this use here, only in the other case.